doc/conf.py

```diff
@@ -71,8 +71,8 @@ def __getattr__(cls, name):
 
 # General information about the project.
 project = u'signac'
-copyright = u'2017, Carl Simon Adorf'
-author = u'Carl Simon Adorf'
+copyright = 'The Regents of the University of Michigan'
+author = u'Carl Simon Adorf, Vyas Ramasubramani, Bradley D. Dice, Michael M. Henry, Paul M. Dodd, Sharon C. Glotzer'
```

Comment:
It says `Carl S. Adorf` on the signac-flow PR. What's your preference, @csadorf?
https://github.com/glotzerlab/signac-flow/pull/102/files#diff-684a0ceb2878d48b19612ca360e277efR68

Reply:
Thx for the catch, I unified it to abbreviated "middle name".